js/index.js

```diff
@@ -1,7 +1,73 @@
 // Iteration 1: Names and Input
+let hacker1 = "Nico";
+console.log(`The driver's name is ${hacker1}`)
 
+let hacker2 = "Natasha"
+console.log(`The navigator's name is ${hacker2}`)
 
 // Iteration 2: Conditionals
 
+if (hacker1.length > hacker2.length) {
+    console.log(`The driver has the longest name, it has ${hacker1.length} characters`)
+} else if (hacker1.length < hacker2.length) {
+    console.log (`It seems that the navigator has the longest name, it has ${hacker2.length} characters`)
+} else {
+    console.log(`Wow, you both have equally long names, ${hacker1.length} characters!.`)
+}
```

Comment:
Buen trabajo.
Solo por ser un poco tiquismiquis lo que podemos hacer es guardar la longitud de las palabras `hacker1` y `hacker2` en variables para que no tengáis que acceder varias veces a esa propiedad.
El código quedaría:
```
const lengthHacker1 = hacker1.lenght;
const lengthHacker2 = hacer2.length;

if (lengthHacker1 > lengthHacker2) {
    console.log(`The driver has the longest name, it has ${lengthHacker1} characters`)
} else if (lengthHacker1 < lengthHacker2) {
    console.log (`It seems that the navigator has the longest name, it has ${lengthHacker2} characters`)
} else {
    console.log (`Wow, you both have equally long names, ${lengthHacker2}  characters!`)
}
```